plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -202,6 +202,12 @@ public interface CoreProperties {
   String PROJECT_TEST_EXCLUSIONS_PROPERTY = "sonar.test.exclusions";
   String GLOBAL_EXCLUSIONS_PROPERTY = "sonar.global.exclusions";
   String GLOBAL_TEST_EXCLUSIONS_PROPERTY = "sonar.global.test.exclusions";
+  
+  /**
+   * @since 9.14
+   */
+  String PROJECT_TESTS_INCLUSIONS_PROPERTY = "sonar.tests.inclusions";
+  String PROJECT_TESTS_EXCLUSIONS_PROPERTY = "sonar.tests.inclusions";
```

Comment:
Typo?

Reply:
I'm not sure I understand, I've followed the typo of other properties. 
But I notice that it can be confusing. 
Would you prefer something like `PROJECT_TEST_INCLUSIONS_ALIAS_PROPERTY`?

Reply:
```suggestion
  String PROJECT_TESTS_EXCLUSIONS_PROPERTY = "sonar.tests.exclusions";
```

Reply:
my bad ...